packages/fast-components-react-msft/src/call-to-action/call-to-action.tsx

```diff
@@ -33,7 +33,7 @@ class CallToAction extends Foundation<
                 {...this.unhandledProps()}
                 className={this.generateClassNames()}
                 disabled={this.props.disabled}
-                href={this.props.href}
+                href={this.props.disabled ? null : this.props.href}
```

Comment:
I think we should always pass `href` to `Button` - behavior between this component and `Button` should be consistent (regarding disabled), so I would just let `Button` handle the semantics/accessibility requirements.

Reply:
I agree with this - I don't think this additional logic is necessary.

Reply:
@nicholasrice Ok, so the button should handle the logic of removing the href if disabled is true?

Currently, if you set an href on a button and then change it to disabled, it adds `aria-disabled=true` and styles the button as disabled, BUT, it is still actionable. This was the issue I was addressing in CTA.

So, if I apply the `href={this.props.disabled ? null : this.props.href}` to button, it would solve this.

Reply:
A disabled anchor is essentially one without an href. If someone passes an href and disabled, I think we want to pass it. Aria by itself doesn't necessarily prevent interaction. This seems to be an issue with styling; we can choose to style it appropriately (cursor behavior) to prevent it being actionable. 

CTA is essentially a style abstraction here; I'm not sure we want to start black boxing behavior such as this to solve an issue that arises from prop combinations someone chooses to pass.

Reply:
> This seems to be an issue with styling; we can choose to style it appropriately (cursor behavior) to prevent it being actionable.

Can you explain this a little more? Currently, button _does_ have the correct styling when disabled and an anchor - the shape is `<a href="#" aria-disabled="true">foo</a>`, BUT it is still actionable (as in, even though it looks like you cannot click it, you can).

I'm fine with saying if a user passes both `disabled` AND an` href`, we will write the shape `<a href="#" aria-disabled="true">foo</a>`, and the user can choose to do whatever. But, out of the box, interaction is not blocked in this case.

So if we are ok with that, I will revert the logic to remove href, and we will just say things that get both disabled and an href will be styled appropriately, but not actually "disabled"

Reply:
I show these both as being actionable and navigating to a page. If you change the actual destination on button from "#" to something like www.microsoft.com, that will navigate you even when disabled

Reply:
Right, that's exactly what I am saying. Because, if either button or CTA has an 'href' prop, they will get `aria-disabled`, but not `disabled` (which actually disables it).

**So**, we are saying the same thing, I just want to make sure this is the _expected_ behavior - in which case, the PR has been updated appropriately.

Reply:
Yeah, I think this is expected. `disabled` is valid for a button, not an href (thus aria-disabled). I think here we are apply correct attribute and semantics for the element (aria-disabled vs disabled) and applying a disabled style.

Reply:
Ok - this is good to go then. Thanks.